VideoApp/VideoApp/ViewControllers/Settings/SDKLogLevel/SDKLogLevelOptionListFactory.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  Copyright (C) 2020 Twilio, Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+class SDKLogLevelOptionListFactory {
+    func makeSDKLogLevelOptionList(
+        title: String,
+        keyPath: ReferenceWritableKeyPath<AppSettingsStoreWriting, SDKLogLevel>
+    ) -> SettingsViewModelSection.Row {
+        .optionList(
+            title: title,
+            selectedOption: AppSettingsStore.shared[keyPath: keyPath].title,
+            viewModelFactory: SelectSDKLogLevelViewModelFactory(title: title, keyPath: keyPath)
+        )
+    }
+}
```

Comment:
Created this factory because SDKLogLevelSettingsViewModel should not be making SelectSDKLogLevelViewModelFactory dependencies directly. It would make the view model untestable. Not that I'm testing this logging code but still like to make it testable.